docs/source/new_project.rst

```diff
@@ -43,7 +43,7 @@ questions:
        code (``src/{{package_name}}``). The package name must start with a lowercase letter, 
        followed by one or more of the following (a-z, 0-9, _).
    * - *What github organization will your project live under?*
-     - This will either be a gihub organization, or your github username, if you're working outside 
+     - This will either be a github organization, or your github username, if you're working outside 
```

Comment:
oh, I thought I fixed that :(

Reply:
You gotta merge https://github.com/lincc-frameworks/python-project-template/pull/423